README.rst

```diff
@@ -128,15 +128,6 @@ For more information about how to do basic analysis, check out the
 
 -----
 
-=======
```

Comment:
I have created a new issue about this problem:
https://github.com/coala/coala-bears/issues/918

Please include `Fixes https://github.com/coala/coala-bears/issues/918` in your commit message